step-api/src/main/java/org/jenkinsci/plugins/workflow/structs/DescribableHelper.java

```diff
@@ -158,20 +163,27 @@ public static Schema schemaFor(Class<?> clazz) {
         private final List<String> mandatoryParameters;
 
         Schema(Class<?> clazz) {
+            this(clazz, null);
+        }
+
+        Schema(Class<?> clazz, Stack<String> tracker) {
             this.type = clazz;
+            if(tracker == null){
+                tracker = new Stack<String>();
+            }
```

Comment:
Added the `@Nonnull` annotation to the Stack in the constructor.  I really like that annotation.